docs/components/InputFieldWrapper/InputFieldWrapper.stories.mdx

```diff
@@ -0,0 +1,45 @@
+import { Meta } from "@storybook/addon-docs";
+
+<Meta title="Components/Private/InputFieldWrapper/Docs" />
+
+# Input Field Wrapper (Internal)
```

Comment:
I would consider removing `(Internal)` now that it's in a Private section, this is implied